app/controlplane/pkg/biz/orgmetrics.go

```diff
@@ -149,6 +160,45 @@ func (uc *OrgMetricsUseCase) TopWorkflowsByRunsCount(ctx context.Context, orgID
 	return uc.repo.TopWorkflowsByRunsCount(ctx, orgUUID, numWorkflows, timeWindow)
 }
 
+// GetLastWorkflowStatusByRun returns the last status of each workflow by its last run
+func (uc *OrgMetricsUseCase) GetLastWorkflowStatusByRun(orgName string) ([]*prometheuscollector.WorkflowLastStatusByRunReport, error) {
```

Comment:
This method should not return another package struct, there is no reason we should have a dependency with prometheuscollector

Reply:
If we add it to the biz package we will run into cycles in the imports that's why it's added on the prometheus package.

Reply:
let me check

Reply:
mm, I cant see cycle. 

```
biz -> prometheus/metrics
prometheus -> biz
```

Reply:
Hmm it maybe it was before when it was not split in different packages. Move it, thanks!

Reply:
thanks for checking it. Something feels wrong here, we'll need to re-evaluate the purposes of those modules later. 

Looks good to me!